docs/index.md

```diff
@@ -440,6 +440,32 @@ No Context
 
 No Context 
 
+### sync-error
+#### Description: A syncronization error has been detected. Inform subscribed clients.
+#### Example: 
+```
+{
+  "context": [
+    {
+      "key": "operationoutcome",
+      "resource": {
+        "resourceType": "OperationOutcome",
+        "issue": [
+          {
+            "severity": "warning",
+            "code": "AppId3456 failed to follow context"
```

Comment:
if this is truly intended to be FHIR, the code needs to be from the valueset. IE: is this transient? Processing?

The details (we should define a valueSet) and diagnostics fields could be used for what is shown in code above.

Reply:
Agree, I just updated with a code of processing and put the narrative in diagnostics